### PR TITLE
get rid of the dependency ch.qos.logback:logback-classic:jar:1.4.8:compile

### PR DESCRIPTION
**Pull request by andythsu**
resolves https://github.com/trinodb/trino-gateway/issues/195

### CODE REVIEW

File: gateway-ha/pom.xml [hunk 1]
```diff
@@ -187,6 +187,12 @@
             <version>1.70</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.codehaus.janino</groupId>
+            <artifactId>janino</artifactId>
+            <version>${dep.janino.version}</version>
+        </dependency>
```
Comment (ebyhr) [inline gateway-ha/pom.xml:194]: I don't feel much benefit to replace the existing dependency with new one. We're going to use airlift as the base plugin that uses JUL for logger.

**Review by ebyhr — CHANGES_REQUESTED** [review]

File: gateway-ha/src/main/resources/logback.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+<!DOCTYPE configuration>
+
+<configuration>
+    <import class="ch.qos.logback.core.ConsoleAppender" />
+    <import class="ch.qos.logback.core.filter.EvaluatorFilter" />
+
+    <appender name="STDOUT" class="ConsoleAppender">
+        <filter class="EvaluatorFilter">
+            <evaluator>
+                <expression>event.getMessage().contains("Authorization")</expression>
```
Comment (ebyhr) [inline gateway-ha/src/main/resources/logback.xml:11]: What happens if we use broken Java expressions here? Does CI catch such mistake?

Comment (andythsu) [reply gateway-ha/src/main/resources/logback.xml:11]: I would assume no. It looks like it print the error message in the console, but CI won't fail

File: gateway-ha/pom.xml (hunk 1 above)
Comment (andythsu) [reply gateway-ha/pom.xml:194]: Just wondering why is trino using JUL for logger? Looks like slf4j is more powerful than JUL

**Comment by mosabua** [comment]
Discussed with @andythsu .. we will just declare the dependency since this is approach is not really helping us improve the overall setup. Andy will send a PR for the dependency declaration.

**Pull request closed without merging**